src/tags/object/AudioNext/view.tsx

```diff
@@ -14,9 +14,9 @@ const AudioNextView: FC<AudioNextProps> = ({ item }) => {
   const [position, setPosition] = useState(1);
   const [audioLength, setAudioLength] = useState(0);
 
-  const [zoom, setZoom] = useState(WS_ZOOM_X.default);
-  const [volume, setVolume] = useState(1);
-  const [speed, setSpeed] = useState(1);
+  const [zoom, setZoom] = useState(Number(item.defaultzoom) ?? WS_ZOOM_X.default);
```

Comment:
Actually looking at these again this will never fallback to anything since `Number()` will produce a value not undefined or null. Probably need to split that out.

Reply:
something like 
```suggestion
  const [zoom, setZoom] = useState(Number(item.defaultzoom ?? WS_ZOOM_X.default));
```

Reply:
even more — `item.defaultzoom` and others will always be strings because of mst. and different places for default values are incosistent, and there is 3 different inconsistencies %)
- `defaultzoom` is 1 in mst, `WS_ZOOM_X.default` in the code; and there is also `WS_ZOOM_X.defaultValue` (no usages)
- `defaultspeed` is 1 in mst, 1 in the code, but there is `SPEED.default` constant
- `defaultvolume` is just 1 in both places